vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Validator.kt

```diff
@@ -489,4 +529,119 @@ class Validator(
         return Verifier.VerifyCredentialResult.SuccessIso(it)
     }
 
+
+    /**
+     * Validate a status list jwt within a relevant context.
+     */
+    fun validateStatusListJwt(
```

Comment:
This method is never called

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b